### PR TITLE
Allow flow filtering for L4 protocols using two ports [BP to 1.6.2]

### PR DESCRIPTION
**Pull request by msherif1234**
Signed-off-by: Mohamed Mahmoud <mmahmoud@redhat.com>
(cherry picked from commit 38031f1830161b98affc29534fb95b5b6286ef3e)

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [ ] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: apis/flowcollector/v1beta1/flowcollector_types.go [hunk 1]
```diff
@@ -211,20 +211,23 @@ type EBPFFlowFilter struct {
...
+	// To filter a range of ports, use a "start-end" range, string format. For example, destPorts: "80-100".
+	// To filter two ports, use a "port1,port2" in string format. For example, `ports: "80,100"`.
 	// +optional
 	DestPorts intstr.IntOrString `json:"destPorts,omitempty"`
 
 	// Ports defines the ports to filter flows by. it can be user for either source or destination ports.
-	// To filter a single port, set a single port as an integer value. For example ports: 80.
-	// To filter a range of ports, use a "start-end" range, string format. For example ports: "80-10
+	// To filter a single port, set a single port as an integer value. For example, ports: 80.
+	// To filter a range of ports, use a "start-end" range, string format. For example, ports: "80-10
```
Comment (stleerh) [inline apis/flowcollector/v1beta1/flowcollector_types.go:229]: Missing a `0".` at the end

File: bundle/manifests/flows.netobserv.io_flowcollectors.yaml [hunk 2]
```diff
@@ -200,8 +201,9 @@ spec:
                             - type: string
                             description: |-
                               Ports defines the ports to filter flows by. it can be user for either source or destination ports.
-                              To filter a single port, set a single port as an integer value. For example ports: 80.
-                              To filter a range of ports, use a "start-end" range, string format. For example ports: "80-10
+                              To filter a single port, set a single port as an integer value. For example, ports: 80.
+                              To filter a range of ports, use a "start-end" range, string format. For example, ports: "80-10
```
Comment (stleerh) [inline bundle/manifests/flows.netobserv.io_flowcollectors.yaml:205]: Same as above on missing a `0".`

File: docs/FlowCollector.md [hunk 3]
```diff
@@ -498,8 +499,9 @@ Example: 10.10.10.10<br/>
         <td>int or string</td>
         <td>
           Ports defines the ports to filter flows by. it can be user for either source or destination ports.
-To filter a single port, set a single port as an integer value. For example ports: 80.
-To filter a range of ports, use a "start-end" range, string format. For example ports: "80-10<br/>
+To filter a single port, set a single port as an integer value. For example, ports: 80.
+To filter a range of ports, use a "start-end" range, string format. For example, ports: "80-10
```
Comment (stleerh) [inline docs/FlowCollector.md:503]: Similar to above

**Review by stleerh — COMMENTED** [review]
Besides the minor comments that got truncated, lgtm.

**Review by jpinsonneau — APPROVED** [review]

**Pull request merged**